conans/client/cmd/linter/export_linter.py

```diff
@@ -79,7 +81,9 @@ def _lint_py3(conanfile_path):
 
 
 def _normal_linter(conanfile_path):
-    args = ["--reports=no", "--disable=no-absolute-import", "--persistent=no", conanfile_path]
+
+    args = ["--reports=no", "--disable=no-absolute-import", "--persistent=no",
+            "--load-plugins=%s" % ",".join(LINTER_PLUGINS), conanfile_path]
```

Comment:
I am not sure how this will work with the external application approach.

Reply:
Mine come first... so figure it out ;)

Reply:
But it looks it should be the same, call the pylint with the --load-plugins parameter pointing to the module.